bamazon.js

```diff
@@ -0,0 +1,87 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+var Table = require('cli-table2');
+
+// creates table
+var table = new Table({
+  head: ["Item ID", "Product", "Department", "Price", "Stock Quantity"],
+  colWidths: [4, 35, 15, 6, 20],
+  chars: { 'top': '═' , 'top-mid': '╤' , 'top-left': '╔' , 'top-right': '╗'
+    , 'bottom': '═' , 'bottom-mid': '╧' , 'bottom-left': '╚' , 'bottom-right': '╝'
+    , 'left': '║' , 'left-mid': '╟' , 'mid': '─' , 'mid-mid': '┼'
+    , 'right': '║' , 'right-mid': '╢' , 'middle': '│' }    
+});
+
+// connect to mySQL
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+  user: "root",
+  password: "root",
+  database: "bamazon_db"
+});
+
+// start the main function when it starts
+connection.connect(function(err) {
+  if (err) throw err;
+  startUp();
+});
+
+// displays table of data
+var startUp = function() {
+	var query = "SELECT * FROM products";
+	connection.query(query, function(err, res) {
+  	for (var i = 0; i < res.length; i++) {
+      table.push(
+        [res[i].item_id, res[i].product_name, res[i].department_name, res[i].price, res[i].stock_quantity]
+      );      
+  	}
+    // creates tables from res values
+    console.log(table.toString());
+    inquire();
+  }) 	
+}
+
+var inquire = function() {
+	inquirer.prompt([
+    {
+      name: "productID",
+      type: "input",
+     	message: "What product ID would you like to choose?"
+    },
+    {
+    	name: "units",
+      type: "input",
+     	message: "How much of this item would you like to purchase?"
+    }
+  ])
+  .then(function(answer) {
```

Comment:
you can save time and make the user wait less if you evaluate your answer with what items are in the table by looking at their answer since you already retrieved all the items for sale from your database earlier. This way you don't have to wait for a specific item retrieval to come back true or false from the database and makes for a smoother UI experience.